eng/pipelines/common/templates/innerloop-steps.yml

```diff
@@ -0,0 +1,15 @@
+parameters:
+  buildConfig: ''
+  testLibrary: ''
+  testRunner: ''
+  condition: always()
+
+steps:
+  # Build and test leaf solution
+  - ${{ if ne(parameters.testLibrary, '') }}:
```

Comment:
this step depends on the testRunner as well... should we check for that as well? 

If we wanted to associate these two parameters as for the same thing we could take an object instead of two parameters, something like:

```yml
- library: name
  testRunner: xunit
```

but that could be in a subsequent PR if needed.

Reply:
The TestRunner property is optional... Agreed, let's make this a bit more flexible a follow-up PR: